add a note about installation via epel for RHEL / CentOS Stream 8 & 9 (#3644)
restic/restic: doc/020_installation.rst

RHEL & CentOS
=============

> For RHEL / CentOS Stream 8 & 9 restic can be installed via epel repository, you can try the following:

Reviewer: ```suggestion
For RHEL / CentOS Stream 8 & 9 restic can be installed from the EPEL repository:
```
